Sources/Preferences/SettingsTabViewController.swift

```diff
@@ -95,9 +95,9 @@ final class SettingsTabViewController: NSViewController, SettingsStyleController
 			} else {
 				let settings: String
 				if #available(macOS 13, *) {
-					settings = Localization[.settings]
+                    			settings = NSLocalizedString("settings", bundle: .module, comment: "")
```

Comment:
Still incorrect indentation.

Reply:
are you using a different indentation configuration? to align, what are yours?!

<img width="491" alt="indentation" src="https://user-images.githubusercontent.com/67648808/185515409-93d5b3b3-e62f-46c0-af77-15b322670344.png">

Reply:
You are asking questions that you can simply check yourself. The existing indentation exists in all the source files.